README.md

```diff
@@ -1,17 +1,74 @@
-# CS220 - Support Tools
+#Support Tools
 
-## Purpose
-A tool which, when given a trie, will print out a visualization of the trie for debugging purposes.
+##Description
+This program allows for the visualization of tries (see https://en.wikipedia.org/wiki/Trie). This tool was developed at the University of Chicago, during Spring Quarter of 2018, for CMSC 22000 - Intro to Software Development, in conjunction with a Redis (https://redis.io/) module for use with tries. This program uses a shell to facillitate interaction between the user and the trie they're visualizing.
 
-## Method
-Given a dictionary stored in a trie, we will use the shell to print out representations of the trie in alphabetical order. There will be four different types of visualizations.
+#Using the shell
+These instructions will guide you through using the Support Tools trie visualization suite.
+
+## Prerequisites
+Make sure you have the most recent master branch of the code from out Github repo(https://github.com/cmsc22000-project-2018/support-tools) !
+You will also need a command line tool, like Terminal, to build and interact with the code.
+
+## Making the code
+Simply open the support-tools directory in a command line tool, switch into the consolidated_shell directory, and type:
+$ make all
+
+## Opening the shell
+From here, simply print ./shell
+
+## Running commands
+The shell greets the user. Type "help" at any point to see a full list of commands!
+
+### Selecting your trie
+Right now, all commands within the shell must indicate the index of one of three hardcoded tries, so 0, 1, or 2. In the future, we will allow for a key-based interaction with a remote Redis server. This document will be updated when such functionality is integrated.
+
+### Visualizing your trie
+The basic format of print commands in the shell is:
+$ print <trie index>
+Running this command alone will print every node in the trie.
+
+By addending "all [prefix]" to the above command, only the nodes flowing down from the entered prefix will be printed.
```

Comment:
Do you mean "append" or "add"? I haven't seen this word used like this. Could just be me though (same for two uses below)

Reply:
I swear I have seem this usage though I don't find it under the dictionary definition of addend! I'll switch it to append.